### DISCUSSION TITLE
milvus1.1.0 insert custom ids

### DISCUSSION THREAD

**Discussion by ingale726** [milvus-io/milvus#21462] [Q&A and General discussion]
How does milvus1.1.0 insert custom ids? Does this version support it?

**Comment by yhmo** [comment]
https://milvus.io/docs/v1.1.1/insert_delete_vector_python.md

Insert a list of vectors. If you do not specify vector IDs, Milvus automatically assigns IDs to the vectors.

**Insert vectors.**
```
vectors = [[random.random() for _ in range(256)] for _ in range(20)]
milvus.insert(collection_name='test01', records=vectors)
```

**You can also specify the vector IDs:**

```
vector_ids = [id for id in range(20)]
milvus.insert(collection_name='test01', records=vectors, ids=vector_ids)
```
